.storybook/main.js

```diff
@@ -9,6 +9,7 @@ module.exports = {
 	stories: [
 		"../apps-rendering/src/**/*.stories.@(js|mdx|ts|tsx)",
 		"../dotcom-rendering/src/**/*.stories.@(tsx)",
+		"../dotcom-rendering/stories/**/*.stories.@(tsx)",
```

Comment:
Should this live in a `generated` subfolder similar to fixtures? I know we don't have manual, but it feels consistent. Or a flatter `generated-stories`?

Reply:
I'd agree giving it a naming scheme that indicates it should only be for generated/automatic stories would be a good idea!